<launch>

    <arg name="pub_topic_position_setpoint" default="/aqc/cmd_pos"/>

    <node pkg="aqc_input_dynamic_reconfigure" type="aqc_input_client_dr_node" name="aqc_input_client_dr_node">
        <remap from="/cmd_position" to="$(arg pub_topic_position_setpoint)"/>
    </node>

    <!-- launch rqt dynamic reconfiguration gui and load default parameters from config file -->
    <!-- <arg name="dr_config_file" default="$(find assignment10)/config/default.yaml"/> -->
    <node name="rqt_reconfigure_gui" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>
    <!-- <node name="reconfigure_load_params" pkg="dynamic_reconfigure" type="dynparam" args="load /$(arg turtle_server_node_name) $(arg dr_config_file)"/> -->

</launch>
